x/upgrade/cli/query.go

```diff
@@ -0,0 +1,58 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/celestiaorg/celestia-app/x/upgrade/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd returns the CLI query commands for this module
+func GetQueryCmd() *cobra.Command {
+	// Group blob queries under a subcommand
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      fmt.Sprintf("Querying commands for the %s module", types.ModuleName),
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(CmdQueryTally())
+
+	return cmd
+}
+
+func CmdQueryTally() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "tally [version]",
```

Comment:
[nit] version is a required argument but the surrounding `[]` make it look like an optional argument.

```suggestion
		Use:   "tally version",
```

IDK if there's an authoritative source for command line argument help syntax. I think `tally <version>` or `tally version` are preferable over `tally [version]`

Ref: 
- https://developers.google.com/style/code-syntax
- https://stackoverflow.com/a/47026846/2813210

Reply:
Yeah good call